src/docs/development/add-to-app/android/add-flutter-fragment.md

```diff
@@ -203,12 +212,13 @@ initial routes (routes other than `/`). To facilitate this, `FlutterFragment`'s
 `Builder` allows you to specify a desired initial route, as shown below:
 
 ```java
-FlutterFragment flutterFragment = new FlutterFragment.Builder()
+// With a new FlutterEngine.
+FlutterFragment flutterFragment = FlutterFragment.withNewEngine()
     .initialRoute("myInitialRoute/")
     .build();
 ```
 
-{{site.alert.note}}
+{{site.alert.note}} 
```

Comment:
I think it's amusing that SO many PRs I see remove trailing white space, but your PRs ADD it. I always figured it was a side effect of whatever editing tool is being used.

Reply:
That is weird. I tried to remove it, hopefully it worked.